src/basic/format-table.c

```diff
@@ -1139,7 +1139,6 @@ int table_print(Table *t, FILE *f) {
                         }
 
                         if (finalize) {
-                                assert(!restart);
                                 break;
                         }
```

Comment:
Please remove unnecessary `{}`.

Reply:
Ah, sorry I forgot it.
But thanks to your comment I noticed this deletion seems not to make any sense, because checking `finalize` has done before reaching L:1142 after changing `break` condition.

I'll revert it.

Reply:
My confusion... Deleting `assert()` is needed to fix https://github.com/systemd/systemd/issues/9549.